crates/invoker-impl/src/lib.rs

```diff
@@ -469,7 +468,7 @@ where
         level = "trace",
         skip_all,
         fields(
-            rpc.service = %service_id.service_name,
+            rpc.service = %invocation_target.service_name(),
```

Comment:
at another place (`InvocationTas::run`) we added `restate.invocation.target = %invocation_target`.